Add Inso CLI on Docker doc (#101)
Kong/insomnia-docs: docs/inso-cli/inso-on-docker.md

---
layout: article-detail
title:  Inso CLI on Docker
category: "Inso CLI"
category-url: inso-cli
---

## Pull `kong/inso`

> To pull the latest `kong/inso` Docker image:

Reviewer: ```suggestion
Pull the latest `kong/inso` Docker image:
```

```shell
docker pull kong/inso:latest
```

All available tags can be found on Inso-CLI's [Docker Hub page](https://hub.docker.com/r/kong/inso/tags).

## Run Inso CLI commands

> To run Insomnia specs in `kong/inso` container, mount the specs folder on your host machine to a `/var/temp` folder on the container. Examples:

Reviewer: ```suggestion
To run Insomnia specs in `kong/inso` container, mount the specs folder on your host machine to a `/var/temp` folder in the container. See the following sections for some examples.
```

> - Mount an Insomnia git sync repository folder to a folder on the container:

Reviewer: ```suggestion
## Examples

### Mount git sync repo

Mount an Insomnia git sync repository folder to a folder in the container:
```

```shell
cd <your-git-sync-repo-folder>

# Run Unit Tests
docker run -it --rm -v $(pwd):/var/temp kong/inso:latest run test -w /var/temp

# Generate Kong Declarative config
docker run -it --rm -v $(pwd):/var/temp kong/inso:latest generate config -w /var/temp -f json
```

> - Mount the Insomnia Application Data folder:

Reviewer: ```suggestion
## Mount Application Data folder 

Mount the Insomnia Application Data folder:
```

```shell

> # On macOS:
> docker run -v $HOME/Library/Application\ Support/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp
> # On Linux:
> docker run -v $HOME/.config/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp
> # On Windows (using Docker for Windows and WSL):
> docker run -v /mnt/c/Users/<your_username>/AppData/Roaming/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp

Reviewer: ```suggestion
# On macOS:
docker run -v $HOME/Library/Application\ Support/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp

# On Linux:
docker run -v $HOME/.config/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp

# On Windows (using Docker for Windows and WSL):
docker run -v /mnt/c/Users/<your_username>/AppData/Roaming/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp
```

```

> - Mount the folder where you keep an Insomnia v4 export:

Reviewer: ```suggestion
### Mount v4 export folder

Mount the folder where you keep an Insomnia v4 export:
```
